docs/src/main/sphinx/optimizer/cost-based-optimizations.rst

```diff
@@ -73,14 +72,38 @@ The valid values are:
 Capping replicated table size
 -----------------------------
 
-Join distribution type will be chosen automatically when join reordering strategy
-is set to ``AUTOMATIC`` or when join distribution type is set to ``AUTOMATIC``.
-In such case it is possible to cap the maximum size of replicated table via
-``join-max-broadcast-table-size`` config property (e.g. ``join-max-broadcast-table-size=100MB``)
-or via ``join_max_broadcast_table_size`` session property (e.g. ``set session join_max_broadcast_table_size='100MB';``)
-This allows to improve cluster concurrency and to prevent bad plans when CBO misestimates size of joined tables.
-
-By default replicated table size is capped to 100MB.
+Join distribution type will be chosen automatically when join reordering
+strategy is set to ``AUTOMATIC`` or when join distribution type is set to
+``AUTOMATIC``. In such cases, it is possible to cap the maximum size of the
+replicated table via the ``join-max-broadcast-table-size`` configuration
+property (e.g. ``join-max-broadcast-table-size=100MB``) or via the
+``join_max_broadcast_table_size`` session property (e.g.
+``set session join_max_broadcast_table_size='100MB';``). This allows you to
+improve cluster concurrency and to prevent bad plans when CBO misestimates size
+of joined tables.
+
+By default, replicated table size is capped to 100MB.
+
+Syntactic join order
+--------------------
+
+If not using cost-based optimization, Trino will default to syntactic join
+ordering. While there is no formal way to optimize queries for this case, it is
+possible to take advantage of how Trino implements joins to make them more
+performant.
+
+Trino uses in-memory `hash joins <https://en.wikipedia.org/wiki/Hash_join>`_.
```

Comment:
Please remove the link altogether. We should avoid linking to unofficial resources for definitions wherever possible, outside of cases where we link to definitions of proprietary technologies (in which case we must link to a page managed by the owner of that technology).

Hash join isn't a trino term, so if someone doesn't know what it is it's on them to research themselves.